docs/api/en/materials/Material.html

```diff
@@ -334,6 +334,15 @@ <h3>[method:null onBeforeCompile]( [param:Shader shader], [param:WebGLRenderer r
 		Unlike properties, the callback is not supported by [page:Material.clone .clone](), [page:Material.copy .copy]() and [page:Material.toJSON .toJSON]().
 		</p>
 
+		<h3>[method:null onProgramChange]( [param:UniformParams params] )</h3>
+		<p>
+		An optional callback that is executed when the shader program used to render the material is changed, regardless of if the program is fetched from cache or freshly compiled.
+		This can be used to set values of custom uniforms that have been added in onBeforeCompile.
+		</p>
+		<p>
+		Unlike properties, the callback is not supported by [page:Material.clone .clone](), [page:Material.copy .copy]() and [page:Material.toJSON .toJSON]().
```

Comment:
What does "Unlike properties" mean?

Reply:
I copied this line verbatim from the documentation for `onBeforeCompile()`. But it might be clearer if we changed both of them to say just "The callback is not supported but Material.clone()". Should I put it in this PR or leave it to be cleaned up separately?